hoot-services/src/main/java/hoot/services/controllers/job/JobCancellationResource.java

```diff
@@ -88,8 +88,8 @@ public Response process(@Context HttpServletRequest request, JobCancellationPara
             throw new ForbiddenException("HTTP" /* This Parameter required, but will be cleared by ExceptionFilter */);
         } else {
             try {
-                this.externalCommandInterface.terminate(jobIdToCancel);
                 this.jobStatusManager.setCancelled(jobIdToCancel, "Cancelled by user!");
+                this.externalCommandInterface.terminate(jobIdToCancel);
```

Comment:
changed this order so that job is marked cancelled even if the terminate call throws an error